section3/exercises/arrays.js

```diff
@@ -96,21 +105,35 @@ YOU DO:
 Using the variables defined below, write a program that will tell a user if they
 will be able to call an Uber.
 
-The user can call an uber if they have 15% battery remaining, or more. In this case, it doesn't matter 
+The user can call an uber if they have 15% battery remaining, or more. In this case, it doesn't matter
 if the user has a charger at all, or what type.
 The can call an uber if they have a charger and it is a car charger.
 
 */
-var percentBatteryLeft = 12;
+var percentBatteryLeft = 11;
 var hasCharger = true;
 var chargerType = "car";
 
+
 // Write your conditional here
 
+if (percentBatteryLeft >= 15) {
+  //Code executes if percentBatteryLeft is greater than or equal to 15
+  console.log('Can call an uber!')
+} else if (percentBatteryLeft < 15 + true + "car") {
```

Comment:
Hmm I'd revisit this section2 review exercise too. You generally won't see math operations inside a conditional statment (unless you're actually adding Numbers). Here you're adding a `boolean` and `boolean` and a `string`, which isn't doing what you think it is.